test/ndb_api_helpers_test.go

```diff
@@ -604,3 +605,50 @@ func TestGenerateProvisioningRequestReturnsErrorIfSSHKeyIsEmpty(t *testing.T) {
 		}
 	}
 }
+
+func TestGetActionArgumentsByDatabaseType(t *testing.T) {
+	// Test with MySQL database type
+	mysqlArgs, err := v1alpha1.GetActionArgumentsByDatabaseType(v1alpha1.DATABASE_TYPE_MYSQL)
+	if mysqlArgs != nil {
```

Comment:
Could you add the check for expected MYSQL arguments like you have done for postgres arguments below...